Remotion/Web/Development.WebTesting/CoypuElementScopeExtensions.cs

```diff
@@ -40,27 +40,56 @@ public static class CoypuElementScopeExtensions
     /// where the <c>parentScope.Find*()</c> call could not be found.
     /// </summary>
     /// <param name="scope">The <see cref="ElementScope"/> which is asserted to exist.</param>
+    /// <exception cref="WebTestException">If the control cannot be found.</exception>
+    /// <exception cref="WebTestException">If multiple matching controls are found.</exception>
     public static void EnsureExistence ([NotNull] this ElementScope scope)

```

Comment:
This method seems to be called after initially selecting controls through `FindCss` etc. If the selector method did not throw an exception but this one does, it might be a framework failure worthy of a `MissingHtmlException` or `AmbiguousException`.

Reply:
Its true that this may happen. But its also used in the WebTestObjectContext where Failure would be more of a WebTestException. I think we have to check that with MK

Reply:
@Stefan-Ilic iirc exceptions are only thrown during scope synchronization, not while creating the selector. So this would be the place where the exception gets thrown.